TESTING.rst

```diff
@@ -1938,3 +1938,36 @@ On the screen you will see database queries for the given test.
 
 SQL query tracking does not work properly if your test runs subprocesses. Only queries from the main process
 are tracked.
+
+Code Coverage
+=============
+
+Airflow's CI process automatically uploads the code coverage report to codecov.io.
+
+Viewing the Coverage Report Online:
+-----------------------------------
+- For the most recent coverage report of the main branch, visit: https://codecov.io/gh/apache/airflow.
+
+Generating Local Coverage Reports:
+----------------------------------
+If you wish to obtain coverage reports for specific areas of the codebase on your local machine, follow these steps:
+
+   a. Initiate a breeze shell.
```

Comment:
These lists don’t render correctly

https://github.com/astronomer/airflow/blob/expose-cov-locally/TESTING.rst#viewing-the-coverage-report-online

Reply:
Fixed. Thanks!